superset-frontend/webpack.config.js

```diff
@@ -419,11 +419,6 @@ const config = {
           {
             loader: '@svgr/webpack',
             options: {
-              icon: true,
```

Comment:
are the defaults good? do we want to be explicit as in
```javascript
options: {
  // making sure that svgo isn't setting width:1em
  icon: false,
  // don't allow svgo to alter/opitmize the svg 
  svgo: false,
},
```

Reply:
Yeah, good idea. The default for svgo is actually true and I don't think it will break anything in this use case if changed to false, but I'll add in the default for icon with a comment.